ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -60,6 +60,12 @@ interface {{ ethernet_interface }}
    switchport trunk native vlan {{ ethernet_interfaces[ethernet_interface].native_vlan }}
 {%             endif %}
 {%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].phone.vlan is defined and ethernet_interfaces[ethernet_interface].phone.vlan %}
+   switchport phone vlan {{ ethernet_interfaces[ethernet_interface].phone.vlan }}
+{%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].phone.trunk is defined and ethernet_interfaces[ethernet_interface].phone.trunk %}
```

Comment:
Please use `arista.avd.defined` filter. Compared to the builtin is defined test, this test will also test for none and can even test for a specific value.

Reply:
I suspect I forgot about this filter given there are still tests in many of the templates that check if defined and not none.  I address the missing filter, but it may be a good idea to scrub the repo and fix everywhere to promote good form.